packages/shopping/public/api.js

```diff
@@ -3,6 +3,72 @@
 'use strict';
 
 const api = {
+
+  async getUser(userId) {
+    const token = localStorage.getItem('shoppyToken');
+    const url = apiUrl + `/users/${userId}`;
+    return $.ajax({
+      type: 'GET',
+      url: url,
+      headers: {
+        Authorization: `Bearer ${token}`,
+      }
+    }).promise();
+  },
+
+  async updateUser(userId, body) {
+    const token = localStorage.getItem('shoppyToken');
+    const url = apiUrl + `/users/${userId}`;
+    return $.ajax({
+      type: 'PUT',
+      url: url,
+      data: JSON.stringify(body),
+      contentType: 'application/json',
+      headers: {
+        Authorization: `Bearer ${token}`,
+      }
+    }).promise();
+  },
+
+  getOrders() {
+    const userId = localStorage.getItem('shoppyUserId');
+    const token = localStorage.getItem('shoppyToken');
+    const roles = localStorage.getItem('shoppyRoles').split(',');
```

Comment:
How do you stop the user altering `shoppyUserId` and `shoppyRoles` in localStorage?  Apologies if I'm jumping the gun on the WIP! 🤐

Reply:
The token will be invalid if they edit anything, so request will be rejected.